corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -422,6 +422,10 @@
       <script src="{% static 'hqwebapp/js/prepaid_modal.js' %}"></script>
     {% endif %}
 
+    {% if show_status_page %}
+      <script src="https://p3s9fvl6gvhr.statuspage.io/embed/script.js"></script>
```

Comment:
Is this really the official URL for this library? This looks subject to change

Reply:
Yes, this is the official URL for this library.

Reply:
I see. p3s9fvl6gvhr is specifically our project.